datastore/longhorn.go

```diff
@@ -792,6 +792,35 @@ func (s *DataStore) ListVolumesByLabelSelector(selector labels.Selector) (map[st
 	return itemMap, nil
 }
 
+// ListVolumesFollowsGlobalSettingsRO returns an object contains all Volumes
+// that don't apply specific spec and follow the global settings
+func (s *DataStore) ListVolumesFollowsGlobalSettingsRO(followedSettingNames map[string]interface{}) (map[string]*longhorn.Volume, error) {
+	itemMap := make(map[string]*longhorn.Volume)
+
+	followedGlobalSettingsLabels := map[string]string{}
+	for sName := range followedSettingNames {
+		followedGlobalSettingsLabels[types.GetLonghornLabelKey(types.LonghornLabelFollowGlobalSetting)] = sName
```

Comment:
Where/when has `LonghornLabelFollowGlobalSetting`  been updated to volumes?

Reply:
Missed the changes in the mutation webhook part.